setup.py

```diff
@@ -94,6 +98,12 @@ def main():
     else:
         install_requires = install_requires + tensorflow_pypi_dep
 
+    if os.environ.get('DS_NOHOROVOD', ''):
```

Comment:
Can we invert the logic here, and make `horodov` an optional dep ? Thus:
```
if os.environ.get("DS_WITH_HORODOV", ""):
    install_requires = install_requires + horovod_pypi_dep
else:
    install_requires = install_requires
```

Reply:
I already thought about this and also prefer this way. We just wanted to keep it consistent to `DS_NOTENSORFLOW` and `DS_NODSDECODE`. We will change it.